pkg/provisioner/reconciler/claim-reconciler/reconiler.go

```diff
@@ -205,13 +210,23 @@ func (r *ObjectBucketClaimReconciler) handleProvisionClaim(key client.ObjectKey,
 	setObjectBucketName(ob, key)
 	ob.Spec.StorageClassName = obc.Spec.StorageClassName
 	ob.Spec.ClaimRef, err = claimRefForKey(key, r.internalClient)
-	ob.SetFinalizers([]string{finalizer})
 	ob.Spec.ReclaimPolicy = options.ReclaimPolicy
+	ob.SetFinalizers([]string{finalizer})
 
 	if ob, err = createObjectBucket(ob, r.internalClient, r.retryInterval, r.retryTimeout); err != nil {
 		return err
 	}
 
+	ob, err = updateObjectBucketPhase(r.internalClient, ob, v1alpha1.ObjectBucketStatusPhaseBound, r.retryInterval, r.retryTimeout)
```

Comment:
why can't `CreateObjectBucket` above also set the status to be bound, vs. 2 API calls?

Reply:
The `status` subresource can only be updated by the `client.UpdateStatus()` method.  Values in the OB.Status at `Create()` are ignored.

Reply:
ok, I didn't know that...